modules/multi-architecture-enabling-64k-pages.adoc

```diff
@@ -0,0 +1,84 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multi-architecture-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-enabling-64k-pages_{context}"]
+
+= Enabling 64k pages on the {op-system-first} kernel 
+
+You can enable 64k memory page in the {op-system-first} kernel on the 64-bit ARM compute machines in your cluster. The 64k page size kernel specification can be used for large GPU or high memory workloads. This is done using the Machine Config Operator (MCO) which uses a machine config pool to update the kernel To enable 64k page sizes, you must create a machine config pool for AMD64 and a machine config pool for ARM64.
```

Comment:
```suggestion
You can enable 64k memory page in the {op-system-first} kernel on the 64-bit ARM compute machines in your cluster. The 64k page size kernel specification can be used for large GPU or high memory workloads. This is done using the Machine Config Operator (MCO) which uses a machine config pool to update the kernel To enable 64k page sizes, you must dedicate a machine config pool for the ARM64 machines to enable this kernel on.
```

We don't mandatory need another machine config pool for the amd64 machines. We must guarantee a dedicated machine config pool for those arm64 machine that should run the 64k pagesize kernel.

Reply:
Okay gotcha, that makes sense to me!